orc8r/cloud/go/services/eventd/obsidian/handlers/handlers.go

```diff
@@ -253,6 +293,12 @@ const (
 	pathParamEnd         = "end"
 )
 
+func getInitErrorHandler(err error) func(c echo.Context) error {
+	return func(c echo.Context) error {
+		return obsidian.HttpError(fmt.Errorf("initialization Error: %v", err), 500)
```

Comment:
minor nit: we should probably move our error codes to a constants file and use that instead of using 500 here.